### PR TITLE
map 

### PR DESCRIPTION
**Pull request by jordansread**
image

this is for #5. 

Haven't added to viz.yaml because I am thinking we are going to have some conflicts if we are all hitting that.

### CODE REVIEW

File: scripts/process/process_map.R [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  library(maps)
+  map <- maps::map(..., fill=TRUE, plot = FALSE)
+  IDs <- sapply(strsplit(map$names, ":"), function(x) x[1])
+  map.sp <- map2SpatialPolygons(map, IDs=IDs, proj4string=CRS("+proj=longlat +datum=WGS84"))
+  map.sp.t <- spTransform(map.sp, CRS("+proj=laea +lat_0=45 +lon_0=-100 +x_0=0 +y_0=0 +a=6370997 +b=6370997 +units=m +no_defs"))
+  return(map.sp.t)
+}
+
+# create the sp object 
+process.map_states <- function(){
```
Comment (jiwalker-usgs) [inline scripts/process/process_map.R]: this needs the viz argument for the dispatching to work

Comment (jordansread) [reply scripts/process/process_map.R]: whoops, fixing

File: scripts/process/process_map.R [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  alaska <- elide(alaska, shift=c(-2100000, -2500000))
+  row.names(alaska) <- 'alaska'
+  proj4string(alaska) <- proj4string(conus)
+  
+  hawaii <- elide(HI, rotate=-35)
+  hawaii <- elide(hawaii, shift=c(5400000, -1400000))
+  row.names(hawaii) <- 'hawaii'
+  proj4string(hawaii) <- proj4string(conus)
+  
+  rbind(conus, alaska, hawaii, makeUniqueIDs = TRUE)
```
Comment (jiwalker-usgs) [inline scripts/process/process_map.R]: And then this should be written to rds for the handoff to visualize?

Comment (jordansread) [reply scripts/process/process_map.R]: geez, it's like I forgot everything I learned

**Pull request merged**